tools/quasitools/complexity_bam.xml

```diff
@@ -0,0 +1,57 @@
+<tool id="complexity_bam" name="Complexity BAM" version="0.7.0">
+    <requirements>
+        <requirement type="package" version="0.7.0">quasitools</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+
+        ln -f -s ${input_bam.metadata.bam_index} ${input_bam}.bai &&
```

Comment:
can you add single quotes around the dataset path? Just in case someone has whitespace(s) in their name.

Same with the command below. Example

quasitools complexity bam '$ref_file' '$input_bam'  $k -f $filter_size -o output.csv